pom.xml

```diff
@@ -90,7 +90,7 @@
     <version.sslr>1.24.0.633</version.sslr>
     <version.spotless>2.43.0</version.spotless>
     <version.sonarlint>9.8.0.76914</version.sonarlint>
-    <version.analyzer-commons>2.11.0.2861</version.analyzer-commons>
+    <version.analyzer-commons>2.12.0.2964</version.analyzer-commons>
```

Comment:
Should we also update rule metadata in this PR, or do you think we can do it right before the release?